### ISSUE TITLE
Improve verbose output - print what file is being copied before the copy starts

### ISSUE THREAD

**Issue by diamondsw** [ncw/rclone#881]
> What is your rclone version (eg output from rclone -V)
1.34
> Which OS you are using and how many bits (eg Windows 7, 64 bit)
macOS 10.12.1, 64-bit
> Which cloud storage system are you using? (eg Google Drive)
Amazon Drive
> The command you were trying to run (eg rclone copy /tmp remote:tmp)
/usr/local/bin/rclone copy --verbose --stats 0 "/Volumes/Anime" "amazon:/Backup/Anime"
> A log from the command with the -v flag (eg output from rclone -v copy /tmp remote:tmp)
2016/11/13 19:15:20 amazon drive root 'Backup/Anime': Waiting for transfers to finish
2016/11/13 19:35:03 Series/Working!/Season 4/WWW.Working!! - 07.mkv: Copied (new)

When using verbose output (useful from scripts), nothing printed *before* a file is copied, only when it is complete. This can result in extremely long delays on large files during which the user doesn't know what is happening. I checked the complete output for the logged session above, and the file "WWW.Working!! - 07.mkv" was never mentioned prior to that entry. Note the long time (~20min) between the "Waiting for transfers to finish" and the "Copied" note.

**Comment by jschwalbe** [comment]
+1, I've been bugged by this behavior as well.

**Comment by diamondsw** [comment]
Probably more than any other issue I've filed, this one is most important to me. Right now it's impossible to tell what rclone is doing, as the only thing on screen is the *last* thing it did. Hoping this is literally a one-line print statement.

**Comment by ncw** [comment]
Note that if you haven't disabled them the rclone stats show which files are transferring. You can change the frequency of them with the --stats flag. 

Does that help?

**Comment by diamondsw** [comment]
Not really. I use the log for one key reason - to see a summary of what changed. When a job is done I want to be able to review everything that happened - copies, deletes, errors and so forth. The stats view doesn't really work for this as it's periodic and truncates the output. The verbose output is better (although I have to do massive filtering down of its output to get to what I want), but it suffers from the problem mentioned here - nothing is printed when a copy begins, only when it ends. That means when copying large files the output just stops, and you may not know what's going on for half and hour or more.
